### PR TITLE
ShuttleSounds & NewStarlight

### PR DESCRIPTION
**Pull request by AliiTea**
Ports:

https://github.com/tgstation/-tg-station/pull/12748  Thx MSO.  <-- Starlight

https://github.com/tgstation/-tg-station/pull/12743  Thx Xhuis and Cuboos.  <-- Shuttle Sounds

e1619c78-7e8d-11e5-8bd1-87bc1cc3eeac

### CODE REVIEW

**Comment by danhibiki** [comment]
Sure, so long as the starlight doesn't eat up a lot of CPU

**Comment by AliiTea** [comment]
It won't

**Comment by MrsTonedOne** [comment]
old: 

New:

**Comment by Nebyab** [comment]
yeah sure

**Comment by AliiTea** [comment]
thanks @MrStonedOne

**Comment by ghost** [comment]
Hey, /tg/ coder here. I had to change the shuttle sounds manually after I was informed that the sounds I used (from Starbound) could potentially cause a copyright claim. You might want to look into doing the same.

**Comment by AliiTea** [comment]
Already changed em to Cuboos' sounds @Xhuis

**Comment by ghost** [comment]
In that case, great.

File: code/game/turfs/space/space.dm [hunk 1]
```diff
@@ -23,7 +23,7 @@
 	if(config)
 		if(config.starlight)
 			for(var/turf/simulated/T in RANGE_TURFS(1,src)) //RANGE_TURFS is in code\__HELPERS\game.dm
-				SetLuminosity(3)
+				SetLuminosity(5,1.5)
```
Comment (MrsTonedOne) [inline code/game/turfs/space/space.dm:26]: So it seems at /tg/ we might decide to have this be 4,1 and not 5,1.5.

See the updated OP for new pictures. (https://github.com/tgstation/-tg-station/pull/12748)

Decide if you want to re-port the new shit.

gif showing all 3 (old, new1, new2):  https://tgstation13.org/msoshit/starlight.gif

However I'd wait until it's merged to re-port. I plan to make starlight effect some of the "outdoors" areas like solars

**Pull request merged**